### PR TITLE
Replace failing "yarn run cli.." by "react-native.."

### PR DESCRIPTION
**Pull request by Tbaut**
It's failing and "has never worked"

### CODE REVIEW

**Comment by maciejhirsz** [comment]
this basically makes `ios` and `build-rust-ios` equivalent (ditto for android), we should remove one of of the commands entirely IMO.

**Comment by Tbaut** [comment]
right, I kept the most descriptive ones.

**Comment by maciejhirsz** [comment]
We could keep the short ones if we change them to:

```json
"ios": "npm run build-rust-ios && react-native run-ios",
"android": "npm run build-rust-android && react-native run-android",
```

**Comment by Tbaut** [comment]
it makes sense, thanks for the guidance :)

File: package.json [hunk 1]
```diff
@@ -4,10 +4,8 @@
   "private": true,
   "license": "GPL-3.0",
   "scripts": {
-    "build-rust-ios": "cd rust/signer && make ios",
-    "ios": "yarn run build-rust-ios && yarn run cli run-ios",
-    "build-rust-android": "cd rust/signer && make android",
-    "android": "yarn run build-rust-android && yarn run cli run-android",
+    "ios": "npm run build-rust-ios && react-native run-ios",
```
Comment (maciejhirsz) [inline package.json]: This is refering to `build-rust-ios`, but `build-rust-ios` is removed :see_no_evil:.

**Pull request merged**